lib/third_party/analytics.dart

```diff
@@ -15,7 +15,11 @@ enum SignInMethod { gmail, apple }
 
 enum PairingDeviceStatus { qr_scanned, success, failed }
 
-enum PairingDeviceType { repaired, recovered, start }
+enum PairingDeviceType { repaired, recovered, new_account }
```

Comment:
Is it intentional to have **`new_account`** in both `PairingDeviceType` and `DistributedKeyGenType`?

Reply:
Yes, it is